shell-completion/bash/systemd-creds

```diff
@@ -0,0 +1,145 @@
+# shellcheck shell=bash
+# systemd-creds(1) completion                   -*- shell-script -*-
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# This file is part of systemd.
+#
+# systemd is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# systemd is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with systemd; If not, see <https://www.gnu.org/licenses/>.
+
+__contains_word() {
+    local w word=$1; shift
+    for w in "$@"; do
+        [[ $w = "$word" ]] && return
+    done
+}
+
+__get_tpm2_devices() {
+    local i
+    for i in /dev/tpmrm*; do
+        [ -c "$i" ] && printf '%s\n' "$i"
```

Comment:
Is it possible to use `--tpm-device=list` ?

Reply:
Unfortunately no, it doesn't support no-pager/no-legend